analytics/lib/counts.py

```diff
@@ -58,11 +75,15 @@ def process_count_stat(stat, fill_to_time):
 
     currently_filled = currently_filled + timedelta(hours = 1)
     while currently_filled <= fill_to_time:
+        logger.info("START %s  %s  %s" %(stat.property, stat.interval, currently_filled))
         FillState.objects.filter(property = stat.property) \
                      .update(end_time = currently_filled, state = FillState.STARTED)
+        start = time.time()
         do_fill_count_stat_at_hour(stat, currently_filled)
+        end = time.time()
         FillState.objects.filter(property = stat.property).update(state = FillState.DONE)
         currently_filled = currently_filled + timedelta(hours = 1)
+        logger.info("DONE %s %s %2.1f ms" %(stat.property, stat.interval, (end-start)*1000))
```

Comment:
We can print the ms as an int, e.g. int((end-start) * 1000). We don't need to worry about distinguishing at a .1ms granularity, and if we ever store this in a perf table we'll store it as an int. Also, '3ms' rather than '3 ms'.